scripts/initdb_and_run.sh

```diff
@@ -4,17 +4,18 @@ export LOCAL_DUMP="/app/scripts/prod_db_dump.sql"
 
 if [ ! -e /app/.cache/mysql/db.done ]; then
     echo "Initializing DB..."
-    python scripts/manage-db.py -d mysql://balrogadmin:balrogadmin@balrogdb/balrog create
     python scripts/import-db.py
```

Comment:
Nit, import-db.py seems to pull down the prod db dump if it's missing/stale, so the name is a bit misleading.

Reply:
Yeah...probably a good time to improve on that.